package.json

```diff
@@ -15,7 +15,8 @@
     "cmr:start_and_setup": "start-server-and-test cmr:start http-get://localhost:3003/collections.json cmr:setup",
     "cmr:stop": "node setup/stopCmr.js",
     "cmr:reset": "node setup/resetCmr.js",
-    "offline": "serverless offline start"
+    "offline": "serverless offline start",
+    "preinstall": "npx npm-force-resolutions"
```

Comment:
I used is to so I can fix fast-xml-parser manually

Reply:
I am not use why you needed to do a force install for that package. According to the npm, the latest version is 4.4.1 https://www.npmjs.com/package/fast-xml-parser. I don't think we should use force to update vulnerabilities because it might be bypassing some dependency checks that might cause other issues down the line.

Reply:
Yeah force install should not be used to fix vulnerabilities.

Reply:
Fixed, removed force install